docs/tutorials.rst

```diff
@@ -0,0 +1,211 @@
+a valid one.
+
+.. code-block::
+    :lineno-start: 8
+
+    # Request information about the node
+    response = api.get_node_info()
+
+Then we can call the :py:meth:`Iota.get_node_info` method of the API
+object to get some basic info about the node.
+
+.. code-block::
+    :lineno-start: 11
+
+    # Using pprint instead of print for a nicer looking result in the console
+    pprint(response)
+
+Finally, we print out the response. It is important to note, that all API
+methods return a python dictionary. Refer to the method's documentation to
+determine what exactly is there in the response ``dict``. Here for example,
+we could list the ``features`` of the node::
+
+    pprint(response['features'])
+
+Send Data
+---------
+In this example, you will learn how to:
+
+- **Encode data to be stored on the Tangle.**
+- **Generate a random IOTA address that doesn't belong to anyone.**
+- **Create a zero-value transaction with custom payload.**
+- **Send a transaction to the network.**
+
+Code
+~~~~
+.. literalinclude:: ../examples/tutorials/02_send_data.py
+   :linenos:
+
+Discussion
+~~~~~~~~~~
+.. code-block::
+    :lineno-start: 1
+
+    from iota import Iota, TryteString, Address, Tag, ProposedTransaction
+    from pprint import pprint
+
+    # Declare an API object
+    api = Iota('https://nodes.devnet.iota.org:443', testnet=True)
+
+We have seen this part before. Note, that now we import more objects which we
+will use to construct our transaction.
+
+Notice ``testnet=True`` in the argument list of the API instantiation. We
+tell the API directly that we will use the devnet/testnet. By default, the API
+is configured for the mainnet.
+
+.. code-block::
+    :lineno-start: 7
+
+    # Prepare custom data
+    my_data = TryteString.from_unicode('Hello from the Tangle!')
+
+If you read :ref:`Basic Concepts` and :ref:`PyOTA Types`, it shouldn't be a
+surprise to you that most things in IOTA are represented as trytes, that are
+:py:class:`TryteString` in PyOTA.
+
+Here, we encode our message with :py:meth:`TryteString.from_unicode` into
+trytes.
+
+.. code-block::
+    :lineno-start: 10
+
+    # Generate a random address that doesn't have to belong to anyone
+    my_address = Address.random()
+
+To put anything (transactions) on the Tangle, it needs to be associated with
+an address. **Since we will be posting a zero-value transaction, nobody has to
+own this address**, therefore we can use the :py:meth:`TryteString.random` (an
+:py:class:`Address` is just a :py:class:`TryteString` with some additional
+attributes and fixed length) method to generate one.
+
+.. code-block::
+    :lineno-start: 13
+
+    # Tag is optional here
+    my_tag = Tag(b'MY9FIRST9TAG')
+
+To tag our transaction, we might define a custom :py:class:`Tag` object. Notice,
+that the ``b`` means we are creating a bytestring now. Each byte in it is
+interpreted as a tryte, therefore we are restricted to the tryte aphabet.
```

Comment:
```rst
To tag our transaction, we might define a custom :py:class:`Tag` object. Notice,
that the ``b`` means we are passing a `bytestring`_ value instead of a unicode
string. This is so that PyOTA interprets our input as literal trytes, rather than a
unicode string that needs to be encoded into trytes.

When passing a bytestring to a PyOTA class, each byte is interpreted as a tryte;
therefore we are restricted to the `tryte aphabet`_.

.. _bytestring: https://docs.python.org/3/library/stdtypes.html#bytes
.. _tryte alphabet: ???
```

Reply:
well said! :)